photon-lib/src/test/java/org/photonvision/PhotonCameraTest.java

```diff
@@ -49,24 +54,36 @@
 import org.photonvision.jni.TimeSyncClient;
 import org.photonvision.jni.WpilibLoader;
 import org.photonvision.simulation.PhotonCameraSim;
+import org.photonvision.targeting.PhotonPipelineMetadata;
 import org.photonvision.targeting.PhotonPipelineResult;
 
 class PhotonCameraTest {
+    NetworkTableInstance inst;
+
     @BeforeAll
     public static void load_wpilib() {
         WpilibLoader.loadLibraries();
 
-        // See #1574 - test flakey, disabled until we address this
-        assumeTrue(false);
+        // // See #1574 - test flakey, disabled until we address this
+        // assumeTrue(false);
```

Comment:
I have re-enabled this test and am using a local-only NT isntance. hopefully this works better?